src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -156,6 +156,12 @@
     <Field type="ulong" name="id" label="Continuation ID" />
   </Event>
 
+  <Event name="VirtualThreadPinned" category="Java Virtual Machine, Runtime" label="Virtual Thread Pinned" thread="true" stackTrace="true">
+    <Field type="string" name="blockingOperation" label="Blocking operation" />
```

Comment:
The label should be "Blocking Operation" with a capital "O".

Labels use headline-style capitalization. See here for more information: https://docs.oracle.com/en/java/javase/21/docs/api/jdk.jfr/jdk/jfr/Label.html

Reply:
Fixed.